tola/settings/dev.py

```diff
@@ -17,8 +17,13 @@
     "SHOW_TOOLBAR_CALLBACK": lambda request: True,
 }
 
+DEBUG = True
+
 OFFLINE_MODE = True
 INTERNAL_IPS = ('127.0.0.1',)
 
 EMAIL_BACKEND = 'django.core.mail.backends.filebased.EmailBackend'
 EMAIL_FILE_PATH = '/tmp/tola-messages'
+
+TOLA_ACTIVITY_URL = 'http://tolaactivity.dev.com'
+TOLA_TRACK_URL = 'http://tolatrack.dev.com'
```

Comment:
We already have these URLs in the `local.py` and it's imported here, so it's better if you add env variables in the `docker-compose-dev.yml`.

Reply:
Ok, done!